app/views/works/new.html.erb

```diff
@@ -1,31 +1,35 @@
 <h1 class="h2 mb-3">Untitled deposit</h1>
- <div class="d-flex tabbable-panes">
```

Comment:
You may want to ignore whitespace changes in the GH UI. Since this is in rubyland, my editor likes to stick with two-space indents, and we had four-space indents in our ERBs beforehand.